### ISSUE TITLE
Fail to get columns from Access Database with str table name

### ISSUE THREAD

**Issue by rockg** [mkleehammer/pyodbc#641]
### Environment

- Python: 2.7.16
- pyodbc: 4.0.27
- OS: Microsoft Windows 10
- DB: Access Database
- driver: Microsoft Access Driver (*.mdb, *.accdb) 16.00.4513.10000

### Issue
Seems similar to #506. Connecting to an Access database and then getting the columns by passing in a str literal which fails with `TypeError: bad argument type for built-in operation` and in some cases causes the application to crash. Passing a unicode table name seems to work fine. I assume this is a strictly python 2 issue.

```
connection = pyodbc.connect(u'UID=;DBQ=C:\\Testing\\pyodbc_test.accdb;DRIVER=Microsoft Access Driver (*.mdb, *.accdb);PWD=')
cursor = connection.cursor()
cursor.columns(table='Table1')
```

Associated SQL traces:
Passing unicode column name:
```
__init__        5668-228	ENTER SQLColumnsW 
		HSTMT               0x0000000016E04EF0
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x000000000F143210 [      -3] "Table1\ 0"
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
```
Passing str column name:
```
__init__        1bcc-6320	ENTER SQLColumnsW 
		HSTMT               0x0000000016F54EF0
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x00000000033B8B40 [      -3] "(\ 0"
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
```
And I have seen in another scenario with:
```
run_geo         56a4-55a0	ENTER SQLColumnsW 
		HSTMT               0x00000258F8A82620
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
		WCHAR *             0x00000258FFFFFFFF <Bad null-terminated string!>
		SWORD                       -3 
		WCHAR *             0x0000000000000000 <null pointer>
		SWORD                       -3 
```
[pyodbc_test.zip](https://github.com/mkleehammer/pyodbc/files/3796009/pyodbc_test.zip)

I can provide the full trace output if necessary. Also attached is the database.

**Comment by gordthompson** [comment]
> Passing a unicode table name seems to work fine.

Seeing as how Python_2 will ride into the sunset in ~2 months why not just use the workaround of passing a Unicode table name?

**Comment by rockg** [comment]
That is what I chose to do. Wanted to bring it up as it seems like the fix for #506 wasn't fully complete and this provided another test case. I am fine if you want to close given Python 2 sunset.
